src/main.ts

```diff
@@ -1,9 +1,21 @@
 import { createApp } from 'vue';
+import { createRouter, createWebHistory } from 'vue-router';
```

Comment:
Please re-organize the list of the imports, move the component imports to the end of the list, and try to group the same type of imports. It would be more readable.